tell_me_your_secrets/__main__.py

```diff
@@ -206,10 +216,13 @@ def write_results_to_file(self):
             write_df.to_csv(file_name)
             module_logger.info(f'Completed Writing Results to File : {self.output_path}')
 
-    def run_signatures(self, file_path, content):
+    def run_signatures(self, file_path, content) -> Tuple[Optional[str], Optional[str]]:
         for signature in self.signatures:
             match_result = signature.match(file_path, content)
-            if match_result:
+            if match_result.matched_value:
+                if match_result.matched_value in self.whitelisted_strings:
+                    module_logger.debug(f'Matched {match_result.matched_value} but skipping since it is whitelisted')
```

Comment:
Could be useful to have the signature name in the debug message:
```suggestion
                    module_logger.debug(f'Signature {signature.name} matched {match_result.matched_value} but skipping since it is whitelisted')
```

Reply:
Thanks!